dicodile/utils/dictionary.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 from scipy import signal
 
-from .csc import reconstruct
+from .csc import _is_rank1, reconstruct
```

Comment:
I would put `_is_rank1` in this file as it is mostly related to the `dictionary`

Reply:
Makes sense, but it leads to a circular import issue, so maybe leave it as-is for now?